Doc/library/http.client.rst

```diff
@@ -115,6 +115,25 @@ The module provides the following classes:
       The *strict* parameter was removed. HTTP 0.9 style "Simple Responses" are
       no longer supported.
 
+This module provides the following function:
+
+.. function:: parse_headers(fp)
+
+   Parse the headers from a file pointer *fp* representing a HTTP
+   request/response. The file has to be a :class:`BufferedIOBase` reader
+   (i.e. not text) and must provide a valid :rfc:`2822` style header.
+
+   This function returns an instance of :class:`email.message.Message`
```

Comment:
I ran the following code (basically taken from the tests) and got a different return instance of `http.client.HTTPMessage`, which is due to the `_class=HTTPMessage` parameter of `parse_headers`. Is this a case where we should document the `_class` parameter (I don't think so), or update the return class (probably).

```>>> from test.test_httplib import FakeSocket
>>> import io
>>> from http import client
>>> s = FakeSocket("")
>>> f = io.BytesIO(s.data)
>>> f.readline()
b''
>>> message = client.parse_headers(f)
>>> message
<http.client.HTTPMessage object at 0x10a75a1f0>
```